### ISSUE TITLE
App doesn't run and crashes with error while loading shared libraries: libvmaf.so.1: cannot open shared object file: No such file or directory

### ISSUE THREAD

**Issue by nikelborm** [telegramdesktop/tdesktop#27244]
### Steps to reproduce

1. install arch with latest linux-lts kernel, gnome 45.2, wayland, pipewire
2. install telegram desktop from arch extra repo

### Expected behaviour

It runs.

### Actual behaviour

It crashes.
```
$ telegram-desktop 
telegram-desktop: error while loading shared libraries: libvmaf.so.1: cannot open shared object file: No such file or directory
```

### Operating system

Arch Linux x86_64, Kernel: 6.1.68-1-lts, GNOME 45.2

### Version of Telegram Desktop

telegram-desktop 4.12.2-1

### Installation source

Other (unofficial) source

### Crash ID

There is no way to get crash id. It does not even comes to that stage. It does not run

**Comment by nikelborm** [comment]
Also I found a fix.
I wasn't able to find libvmaf package in standard arch repos. But even without being able to find the package I had libvmaf already installed. The problem is that I had installed newest version. But telegram depends on older version.
This is what i had initially:
```
⚡️98% ➜ ll /lib|grep vmaf
0777  lrwxrwxrwx       - root :      libvmaf.so -> libvmaf.so.3*
0777  lrwxrwxrwx       - root :      libvmaf.so.3 -> libvmaf.so.3.0.0*
0755  .rwxr-xr-x   782Ki root :      libvmaf.so.3.0.0*
```

Luckily I was able to find libvmaf.so.3 in AUR. This package fixes the problem: https://aur.archlinux.org/packages/libvmaf-compat

But obviosly something still needs to be updated in telegram
